lms/djangoapps/mobile_api/users/views.py

```diff
@@ -183,8 +192,22 @@ def get(self, request, course, *args, **kwargs):
         """
         Get the ID of the module that the specified user last visited in the specified course.
         """
+        user_status = self._get_course_info(request, course)
 
-        return self._get_course_info(request, course)
+        api_version = self.kwargs.get("api_version")
+        if api_version == API_V1:
+
+            """
+            Get ID of the block that the specified user last visited in the specified course.
+            """
```

Comment:
Change this to a single line comment.
```suggestion
           # Get ID of the block that the specified user last visited in the specified course.
```